src/app/space/create/deployments/services/deployments.service.spec.ts

```diff
@@ -1381,7 +1387,25 @@ describe('DeploymentsService', () => {
     });
   });
 
-  describe('HTTP error handling', () => {
+  // TODO: re-enable error propagation.
+  // See https://github.com/openshiftio/openshift.io/issues/2360#issuecomment-368915994
+  xdescribe('HTTP error handling', () => {
```

Comment:
I think this needs to be changed back to `describe`.

Reply:
Oh nevermind. I think I misunderstood, it's disabled till the TODO is done.

Reply:
Er actually, maybe it should be enabled to run the test you added. Then add xit to the one below? Anyways LGTM after you resolve this.

Reply:
All three of the tests time out because of the change from `Observable.throw` to `Observable.empty` - since the error is now simply swallowed, the test suite's subscription simply hangs, and there's no detectable event for the tests to handle. Each of the three tests simply checks that a specific type of error handling is performed (passing to global error handler, passing to logger, and sending notification), so all three see the same events or lack thereof.

Reply:
Okay, thanks! Makes sense now :)